src/backend/backconfig.c

```diff
@@ -110,7 +112,6 @@ void out_config_init(
         config.flags |= CFGalwaysframe; // PIC needs a frame for TLS fixups
     }
     config.objfmt = OBJ_ELF;
-    config.ehmethod = dwarfeh ? EH_DWARF : EH_DM;
```

Comment:
That's exactly what it means, and one reason I did it that way. Another reason is it makes it much easier to test, because you cannot build druntime with a non-functional EH system.